tests/ocf-url_origin/EPUB/package.opf

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Open Container Format</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:date>2021-03-05</dc:date>
+    <dc:description>The origin value is displayed. The origin must be unique for each user-specific instance of the publication.</dc:description>
+    <dc:identifier id="pub-id">ocf-url_origin</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>Check origin value</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#sec-container-iri-origin</meta>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#confreq-rs-scripted-origin-2</meta>
```

Comment:
Nit: can we change this anchor? I don't love "-2"; I think anchors should be descriptive.

Perhaps #confreq-rs-scripted-origin-user (formerly -2) and #confreq-rs-scripted-origin-shared (formerly -1)?

Reply:
yep